tools/rp_storage_tool/src/bucket_reader.rs

```diff
@@ -1821,7 +1869,7 @@ impl BucketReader {
 
             debug!("Storing lifecycle marker for {} from key {}", ntr, key);
 
-            if let Some(_) = self.lifecycle_markers.insert(ntr, marker) {
+            if let Some(_) = self.lifecycle_markers.insert(labeled_ntr, marker) {
```

Comment:
```suggestion
            if self.lifecycle_markers.insert(labeled_ntr, marker).is_some() {
```

nit: this construct wasn't added in this PR but might be good to make this a little cleaner

Reply:
Done